docs/getting-started/quick-start.md

```diff
@@ -3,28 +3,70 @@ category: installation
+CKEditor&nbsp;5 is a powerful, rich text editor you can embed in your web application. This guide will show you the fastest way to start using it.
 
-<span class="navigation-hint_mobile">
-	<info-box>
-		Use the **main menu button in the upper-left corner** to navigate through the documentation.
-	</info-box>
-</span>
+## Try CKEditor&nbsp;5 builder
 
-<!-- the wrapper class should be transformed into an additional infobox class if we will want to use it on other pages-->
-<!-- <div class="info-cta"> -->
-<info-box>
-	**Migrating from CKEditor 4?** {@link updating/migration-from-ckeditor-4 Check the migration section}
-</info-box>
+Check out our interactive builder to quickly get a taste of CKEditor&nbsp;5. It offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs.
+
+## Installing CKEditor&nbsp;5 using npm
 
 <info-box>
-	**Want to remove the "Powered by CKEditor" logo?** {@link support/managing-ckeditor-logo Check the dedicated guide}
+
+**Prerequisites:**
+
+* [Node.js](https://nodejs.org/) in version 18 or above.
+* An initialized Node.js project.
+* Basic familiarity with a terminal.
+
 </info-box>
-<!-- </div> -->
 
-TODO
+Before running any command, make sure that your Node.js version is up-to-date. Check if you are in the proper working directory with the initialized Node.js project. Then, run the following command:
+
+```bash
+npm install ckeditor5
+```
+
+The command will install the main CKEditor&nbsp;5 open-source package containing essential plugins.
+
+After that, you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately. Importing and registering UI translations is optional.
+
+```js
+import { ClassicEditor, Essentials, Bold, Italic, Heading, Paragraph } from 'ckeditor5';
+import translations from 'ckeditor5/dist/translations/en.js';
+
+import 'ckeditor5/dist/styles.css';
+
+await ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ Essentials, Bold, Italic, Heading, Paragraph ],
+		toolbar: {
+			items: [ 'undo', 'redo', '|', 'heading', '|', 'bold', 'italic' ]
+		},
+		translations
+	} )
+	.catch( err => {
+		console.error( err );
+	} );
+```
+
+Pass the imported plugins inside the configuration to the {@link module:editor-classic/classiceditor~ClassicEditor#create `create()`} method. The first argument in this function is a DOM element for the editor placement, so you need to add it to your HTML page.
+
+```html
+<div id="editor">
+	<h1>Hello from CKEditor 5!</h1>
+</div>
+```
+
+That is all the code you need to see a bare-bone editor running in a web browser.
+
```

Comment:
I think we need to create a section with adding some premium plugins after this fast start.

`npm install ckeditor5-premium-features`, and some example plugin, and licenseKey.

Reply:
That solves the problem of one complicated code snippet. Good idea.